libs/components/theme/src/lib/styles/themes/modern/_elevation.scss

```diff
@@ -36,8 +36,7 @@
   }
 
   .sky-elevation-24 {
-    box-shadow: $sky-theme-modern-elevation-24-shadow-size
-      $sky-theme-modern-elevation-24-shadow-color;
+    box-shadow: var(--sky-elevation-overlay-400);
```

Comment:
We use this class in toast - updated here to allow us to tokenize when we did this component.